pybaseball/utils.py

```diff
@@ -40,6 +40,22 @@
                     'WIL': 1884, 'WOR': 1880, 'WSA': 1961, 'WSH': 1901,
                     'WSN': 2005}
 
+STATCAST_VALID_DATES = {
+    2008: {'min_valid_date': '2008-03-25', 'max_valid_date': '2008-10-27'},
+    2009: {'min_valid_date': '2009-04-05', 'max_valid_date': '2009-11-04'},
+    2010: {'min_valid_date': '2010-04-04', 'max_valid_date': '2010-11-01'},
+    2011: {'min_valid_date': '2011-03-31', 'max_valid_date': '2011-10-28'},
+    2012: {'min_valid_date': '2012-03-28', 'max_valid_date': '2012-10-28'},
+    2013: {'min_valid_date': '2013-03-31', 'max_valid_date': '2013-10-30'},
+    2014: {'min_valid_date': '2014-03-22', 'max_valid_date': '2014-10-29'},
+    2015: {'min_valid_date': '2015-04-05', 'max_valid_date': '2015-11-01'},
+    2016: {'min_valid_date': '2016-04-03', 'max_valid_date': '2016-11-02'},
+    2017: {'min_valid_date': '2017-04-02', 'max_valid_date': '2017-11-01'},
+    2018: {'min_valid_date': '2018-03-29', 'max_valid_date': '2018-10-28'},
+    2019: {'min_valid_date': '2019-03-20', 'max_valid_date': '2019-10-30'},
+    2020: {'min_valid_date': '2020-07-23', 'max_valid_date': '2020-11-15'}
```

Comment:
Make max valid for this year like 10-30 or something?

Reply:
I figured we'd update it when the world series is over

Reply:
Eh, let's go for a somewhat tighter upper bound (and now we should know to within a day really).